db/migrate/20160831015641_remove_has_muted_from_users.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveHasMutedFromUsers < ActiveRecord::Migration
+  def change
+    User.where(has_muted: true).each do |user|
+      user.experienced!("threadMuted")
```

Comment:
yeah.. I think your preference sounds better too.

On Wed, 31 Aug 2016 at 21:16 HSalmon <notifications@github.com> wrote:

> In db/migrate/20160831015641_remove_has_muted_from_users.rb
> <https://github.com/loomio/loomio/pull/3633#discussion_r76922085>:
>
> > @@ -0,0 +1,8 @@
> > +class RemoveHasMutedFromUsers < ActiveRecord::Migration
> > +  def change
> > +    User.where(has_muted: true).each do |user|
> > +      user.experienced!("threadMuted")
>
> Happy to change, was just thinking about how you would say it aloud, e.g. *user
> experienced mutingThread*
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/loomio/loomio/pull/3633/files/e537f4912b190ca14986bfbb8719eca2217a272f..3d2c46a162a2f18259767f717e05b217acb81ed2#r76922085>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAdr34pv3NxZx2S-HOQIcaD1NtRY_LGOks5qlQBdgaJpZM4Jl_1o>
> .
>